src/sagemaker/accept_types.py

```diff
@@ -0,0 +1,103 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""This module is for SageMaker accept types."""
+from __future__ import absolute_import
+from typing import List
+
+from sagemaker.jumpstart import artifacts, utils as jumpstart_utils
+
+
+def retrieve(
```

Comment:
Let's discuss in standup please, I think that we should have called `instance_types.retrieve` `retrieve_options`, because it is confusing with the `image_url.retrieve` that returns a single value, not a list.

Essentially, not thrilled with maintaining consistency with an inconsistent precedent.